translations/translations.json

```diff
@@ -3461,7 +3461,7 @@
       "notAvailable": "Standortbestimmung deaktiviert",
       "timeout": "Standortbestimmung fehlgeschlagen",
       "refresh": "Liste der nahegelegenen Regionen aktualisieren",
-      "pageTitle": "Regionen",
+      "pageTitle": "Lokale Informationen für Dich",
```

Comment:
do I have to delete the translations for the other language, so that this will be translated?

Reply:
Not sure, I always delete them but if you want to know for sure best ask the service team :)